### ISSUE TITLE
TCC.xsl:42-44: TCC: inclusion of inherited attributes and...

### ISSUE THREAD

**Issue by 0pdd** [yegor256/jpeek#193]
The puzzle `120-5199fe22` from #120 has to be resolved:

https://github.com/yegor256/jpeek/blob/d73541f09fb419892636db79c043182c309ce06f/src/main/resources/org/jpeek/metrics/TCC.xsl#L42-L44

The puzzle was created by rultor on 17-Feb-18. 

Estimate: 30 minutes, role: IMP.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#193](https://github.com/yegor256/jpeek/issues/193) is now in scope, role is `DEV`

**Comment by 0pdd** [comment]
@0pdd all puzzles are solved

**Comment by 0pdd** [comment]
@0pdd all puzzles are solved

**Comment by 0pdd** [comment]
@0pdd all puzzles are solved

**Comment by 0crat** [comment]
The job [#193](https://github.com/yegor256/jpeek/issues/193) assigned to @gauee[/z](https://www.0crat.com/u/gauee), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/1870727249); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
@gauee[/z](https://www.0crat.com/u/gauee) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by gauee** [comment]
the issue depends on #187 which has not been finished yet.
please wait

**Comment by gauee** [comment]
wait

**Comment by 0crat** [comment]
The user @gauee[/z](https://www.0crat.com/u/gauee) resigned from [#193](https://github.com/yegor256/jpeek/issues/193), please stop working. Reason for job resignation: It is older than 10 days, see [§8](http://www.zerocracy.com/policy.html#8)

**Comment by 0crat** [comment]
Resigned on delay, see [§8](http://www.zerocracy.com/policy.html#8): -30 point(s) just awarded to @gauee[/z](https://www.0crat.com/u/gauee)

**Comment by gauee** [comment]
@yegor256, could you please reassign that issue to me.

**Comment by gauee** [comment]
@0crat wait until #187 is fixed, as the puzzle mentions

**Comment by 0crat** [comment]
> @0crat wait until #187 is fixed, as the puzzle mentions ([here](https://github.com/yegor256/jpeek/issues/193#issuecomment-405183810))

@gauee Job [#193](https://github.com/yegor256/jpeek/issues/193) is not assigned, can't put it on hold

**Comment by yegor256** [comment]
@0crat assign @gauee

**Comment by 0crat** [comment]
> @0crat assign @gauee  ([here](https://github.com/yegor256/jpeek/issues/193#issuecomment-405498572))

@yegor256 The job [#193](https://github.com/yegor256/jpeek/issues/193) assigned to @gauee[/z](https://www.0crat.com/u/gauee), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/1980824189); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The user @gauee[/z](https://www.0crat.com/u/gauee) resigned from [#193](https://github.com/yegor256/jpeek/issues/193), please stop working. Reason for job resignation: It is older than 10 days, see [§8](http://www.zerocracy.com/policy.html#8)

**Comment by 0crat** [comment]
Resigned on delay, see [§8](http://www.zerocracy.com/policy.html#8): -30 point(s) just awarded to @gauee[/z](https://www.0crat.com/u/gauee)

**Comment by 0crat** [comment]
The job [#193](https://github.com/yegor256/jpeek/issues/193) assigned to @serranya[/z](https://www.0crat.com/u/serranya), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/bdcd47d8-f04f-4d36-8a83-8827b3d525c0); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by Serranya** [comment]
@0crat wait for #187

**Comment by 0crat** [comment]
> @0crat wait for #187 ([here](https://github.com/yegor256/jpeek/issues/193#issuecomment-443659207))

@Serranya The impediment for [#193](https://github.com/yegor256/jpeek/issues/193) was registered successfully by @serranya[/z](https://www.0crat.com/u/serranya)

**Comment by Serranya** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/jpeek/issues/193#issuecomment-466074530))

@Serranya The user @serranya[/z](https://www.0crat.com/u/serranya) resigned from [#193](https://github.com/yegor256/jpeek/issues/193), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 point(s) just awarded to @serranya[/z](https://www.0crat.com/u/serranya)

**Comment by 0crat** [comment]
The job [#193](https://github.com/yegor256/jpeek/issues/193) assigned to @danielpalafox[/z](https://www.0crat.com/u/danielpalafox), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/630321ec-fc4f-49a2-ba0a-9c8f4eb6a6cb); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by DanielPalafox** [comment]
@0crat wait

**Comment by 0crat** [comment]
> @0crat wait  ([here](https://github.com/yegor256/jpeek/issues/193#issuecomment-475909345))

@DanielPalafox The impediment for [#193](https://github.com/yegor256/jpeek/issues/193) was registered successfully by @danielpalafox[/z](https://www.0crat.com/u/danielpalafox)
